### DISCUSSION TITLE
[3.8.6] Undefined symbol: Blackboard.getAny()

### DISCUSSION THREAD

**Discussion by MHantzsch** [BehaviorTree/BehaviorTree.CPP#753] [Q&A]
I can build our program with the BehaviorTree.CPP version 3.8.6. But upon running the program it throws the following error: 

undefined symbol: _ZNK2BT10Blackboard6getAnyERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE

The error does not occur when i use version 3.8.5 or below.

**Comment by facontidavide** [comment]
unless that program is **Nav2** or technical support is contracted, I do not support version 3.X.

Please provide a sample project that reproduces the error and I may have a look.

**Comment by facontidavide** [comment]
Moving this to Discussions, instead of issues, until you show that there is a problem with the library or the CMakeLists somehow.
